code/source/main.cpp

```diff
@@ -1,7 +1,11 @@
 #include <iostream>
 
+#include "util.cpp"
+
 int main() {
 	std::cout << "Hello World!" << std::endl;
+
+	helloWorld();
```

Comment:
this function is named badly yo

```suggestion
	fooBar();
```